README.md

```diff
@@ -1195,6 +1196,32 @@ datetime                           fromAddress  fromAddressInExchange
 2019-04-30 15:17:28+00:00  0x876eabf441b2ee...                   True  0x1f4a90043cf2d...                False  0xc85892b9ef8c64...   20544.42975
 ```
 
+### Get Metric
```

Comment:
@IvanIvanoff @kalkov Do you have Idea how to better present these metrics in the README. Now it looks to me like its just another metric but its a pretty big feature.

Reply:
@tspenov For sure we need to have a list of all metrics here(first function argument)